sonar-application/src/main/java/org/sonar/application/App.java

```diff
@@ -56,7 +56,7 @@ public void start(Props props) {
     if (props.valueAsBoolean(ProcessConstants.ENABLE_STOP_COMMAND, false)) {
       // stop application when file <temp>/app.stop is created
       File tempDir = props.nonNullValueAsFile(ProcessConstants.PATH_TEMP);
-      ProcessCommands commands = new ProcessCommands(tempDir, "app");
+      ProcessCommands commands = new ProcessCommands(tempDir, 1);
```

Comment:
Index is an internal information used only by process monitor. I think indices should be hidden from caller and should automatically generated and incremented in Monitor#start().